docs/cse/records-signals-entities-insights/view-manage-entities.md

```diff
@@ -171,16 +171,16 @@ Note that:
 id,suppressed,criticality,tags,tags_to_add,tags_to_remove
 _ip-zone1-10.0.0.5,false,default,,Office-Based,
 _ip-zone1-10.0.0.6,true,default,,Office-Based,Remote
```

Comment:
@davidsnyder-jask @MariePMcGarry are the extra commas intentional?

Reply:
yes, @kimsauce. That's how you supply a blank value.